### PR TITLE
[WIP] Format js

### PR DESCRIPTION
**Pull request by davidbarna**
Format code with new sui-lint rules.
This is used to validate new rules.
Build will fail until new version of sui-lint is published.
The new rules can be reviewed here: https://github.com/SUI-Components/sui/pull/278

### CODE REVIEW

File: components/ad/smartbanner/src/index.js [hunk 1]
```diff
@@ -1,42 +1,66 @@
...
+    const {
+      imageUrl,
+      title,
+      text,
+      buttonText,
+      staticPosition,
+      icon: IconClose,
+      ratingValue,
+      ratingMax,
+      customRatingIcons
```
Comment (jcger) [inline components/ad/smartbanner/src/index.js:30]: could we add the trailing comma?

Comment (davidbarna) [reply components/ad/smartbanner/src/index.js:30]: I maintain the already approved prettier config:
https://github.com/SUI-Components/sui/blob/master/packages/sui-lint/.prettierrc.js#L5

But let's see:
we had `only-multiline` https://github.com/SUI-Components/sui/compare/0566e646292bb7e1d351e81e46ee237d43dc5373...5a839108405b5bcaa4f3d337d23de5d2241ac6b9#diff-d1bf7adb61d3977be2d4e6cc1b7f9fabL141

We can change it, but we have to choose between `es5` and `all`
https://prettier.io/docs/en/options.html#trailing-commas

I vote all.
What do you think @carlosvillu @miduga @jcger ?

Comment (jcger) [reply components/ad/smartbanner/src/index.js:30]: all

Comment (carlosvillu) [reply components/ad/smartbanner/src/index.js:30]: I don't know, which are the differences between es6 and ALL

Comment (carlosvillu) [reply components/ad/smartbanner/src/index.js:30]: But +1 for trailing commas !!!

Comment (davidbarna) [reply components/ad/smartbanner/src/index.js:30]: We already decided not to use it with Julian.

Comment (davidbarna) [reply components/ad/smartbanner/src/index.js:30]: Anyways it's late. It's been 6 days and we agreed to merge and repos were reformatted. 😩

**Pull request closed without merging**